src/main/java/org/jabref/gui/actions/MassSetFieldAction.java

```diff
@@ -235,6 +251,10 @@ public void actionPerformed(ActionEvent e) {
                 ce.addEdit(MassSetFieldAction.massRenameField(entryList, fields[0], renameTo.getText(),
                         overwrite.isSelected()));
             }
+        } else if (append.isSelected()) {
+            for (String field1 : fields) {
```

Comment:
Nitpick: `field1` is not really a good variable name here. The loop iterates over all fields and not just the first. Please rename this to `field`.